rapyuta_io_sdk_v2/async_client.py

```diff
@@ -0,0 +1,1449 @@
+        """Delete a project by its GUID.
+
+        Args:
+            project_guid (str): Project GUID
+
+        Returns:
+            Munch: Project details as a Munch object.
+        """
+
+        return await self.c.delete(
+            url=f"{self.v2api_host}/v2/projects/{project_guid}/",
+            headers=self.config.get_headers(with_project=False, **kwargs),
+        )
+
+    @handle_and_munchify_response
+    async def update_project_owner(
+        self, body: dict, project_guid: str = None, **kwargs
+    ) -> Munch:
+        """Update the owner of a project by its GUID.
+
+        Returns:
+            Munch: Project details as a Munch object.
+        """
+        project_guid = project_guid or self.config.project_guid
+
+        return await self.c.put(
+            url=f"{self.v2api_host}/v2/projects/{project_guid}/owner/",
+            headers=self.config.get_headers(**kwargs),
+            json=body,
+        )
+
+    # -------------------Package-------------------
+    @handle_and_munchify_response
+    async def list_packages(
+        self,
+        cont: int = 0,
+        limit: int = 50,
+        label_selector: list[str] = None,
+        name: str = None,
+        regions: list[str] = None,
+        **kwargs,
+    ) -> Munch:
+        """List all packages in a project.
+
+        Args:
+            cont (int, optional): Start index of packages. Defaults to 0.
+            limit (int, optional): Number of packages to list. Defaults to 50.
+            label_selector (list[str], optional): Define labelSelector to get packages from. Defaults to None.
+            name (str, optional): Define name to get packages from. Defaults to None.
+            regions (list[str], optional): Define regions to get packages from. Defaults to None.
+
+        Returns:
+            Munch: List of packages as a Munch object.
+        """
+
+        return await self.c.get(
+            url=f"{self.v2api_host}/v2/packages/",
+            headers=self.config.get_headers(**kwargs),
+            params={
+                "continue": cont,
+                "limit": limit,
+                "labelSelector": label_selector,
+                "name": name,
+                "regions": regions,
+            },
+        )
+
+    @handle_and_munchify_response
+    async def create_package(self, body: dict, **kwargs) -> Munch:
+        """Create a new package.
+
+        The Payload is the JSON format of the Package Manifest.
+        For a documented example, run the rio explain package command.
+
+        Returns:
+            Munch: Package details as a Munch object.
+        """
+
+        return await self.c.post(
+            url=f"{self.v2api_host}/v2/packages/",
+            headers=self.config.get_headers(**kwargs),
+            json=body,
+        )
+
+    @handle_and_munchify_response
+    async def get_package(
+        self, name: str, project_guid: str = None, version: str = None, **kwargs
```

Comment:
todo: Here the project_guid can go into kwargs as its not required.
```suggestion
        self, name: str, version: str = None, **kwargs
```